src/features/game/types/craftables.ts

```diff
@@ -991,6 +991,7 @@ export const ANIMALS: () => Record<Animal, CraftableItem> = () => ({
 /**
  * getKeys is a ref to Object.keys, but the return is typed literally.
  */
+// TODO: Dangerous function that returns array of strings, but acts like returns array of T
```

Comment:
TS wont catch errors if T is number how it was in levels. For example if we take the key(level) from such object - {30: 22}, and further in code we will decide to get level+1 we will get 301, instead of 31